docs/integrations/hosts-operating-systems/opentelemetry/opentelemetry-collector-insights.md

```diff
@@ -0,0 +1,447 @@
+2. Place Env file in the following directory:
+  ```sh
+  /etc/otelcol-sumo/env/
+  ```
+3. Restart the collector using:
+  ```sh
+  sudo systemctl restart otelcol-sumo
+  ```
+
+</TabItem>
+<TabItem value="Windows">
+
+1. Add the telemetry configuration to your existing collector configuration file in `C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config\conf.d` or the main configuration file.
+2. Restart the collector using:
+  ```sh
+  Restart-Service -Name OtelcolSumo
+  ```
+
+</TabItem>
+<TabItem value="macOS">
+
+1. Add the telemetry configuration to your existing collector configuration file in `/etc/otelcol-sumo/conf.d/` or the main configuration file.
+2. Restart the otelcol-sumo process using:
+  ```sh
+  otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
+  ```
+
+</TabItem>
+<TabItem value="Chef">
+
+import ChefEnv from '../../../reuse/apps/opentelemetry/chef-with-env.md';
+
+<ChefEnv/>
+
+</TabItem>
+
+<TabItem value="Ansible">
+
+import AnsEnv from '../../../reuse/apps/opentelemetry/ansible-with-env.md';
+
+<AnsEnv/>
+
+</TabItem>
+
+<TabItem value="Puppet">
+
+import PuppetEnv from '../../../reuse/apps/opentelemetry/puppet-with-env.md';
+
+<PuppetEnv/>
+
+</TabItem>
+</Tabs>
+
+import LogsOutro from '../../../reuse/apps/opentelemetry/send-logs-outro.md';
+
+<LogsOutro/>
+
+### Validation
+
+After installation, verify that:
+1. The OTEL Collector service is running.
+2. The configured base endpoint is reachable.
+3. Data is being successfully sent to both the logs (`/v1/logs`) and metrics (`/v1/metrics`) endpoints.
+4. Resource attributes are properly applied to the telemetry data.
+5. Internal metrics are accessible at `http://localhost:8888/metrics`.
+
+## Sample log messages
+
+```json
+{
+  "timestamp": "2024-01-15T10:30:45.123Z",
+  "level": "info",
+  "msg": "Batch processor started",
+  "component": "batch",
+  "pipeline": "metrics"
+}
+```
+
+```json
+{
+  "timestamp": "2024-01-15T10:30:46.456Z",
+  "level": "warn",
+  "msg": "Dropping data because sending_queue is full",
+  "component": "sumologicexporter",
+  "pipeline": "logs"
+}
+```
+
+## Sample metrics
+
+```json
+{
+  "metric": "otelcol_processor_batch_batch_send_size",
+  "sumo.datasource": "otel_collector",
+  "_contentType": "OpenTelemetry",
+  "deployment.environment": "production",
+  "processor": "batch",
+  "value": 100,
+  "timestamp": "2024-01-15T10:30:45.123Z"
+}
+```
+
+```json
+{
+  "metric": "otelcol_exporter_queue_size",
+  "sumo.datasource": "otel_collector",
+  "_contentType": "OpenTelemetry",
+  "deployment.environment": "production",
+  "exporter": "sumologic",
+  "value": 150,
+  "timestamp": "2024-01-15T10:30:45.123Z"
+}
+```
+
+## Key Internal Metrics
+
+The OpenTelemetry Collector emits comprehensive internal metrics categorized by verbosity levels:
```

Comment:
Can we redirect to link instead of listing all the metric here. The list might change in the future